force-app/main/default/classes/SOQL.cls

```diff
@@ -40,8 +42,11 @@ public with sharing class SOQL {
      * Update the corresponding test class and testGetContactsByFirstName method to cover this method
      */
     //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Contact> getContactsByFirstName() {
+        //List<Contact> conList = [SELECT Name FROM Contact WHERE FirstName = 'John'];
+        //return conList;
+        return (List<Contact>)[SELECT Name FROM Contact WHERE FirstName = 'John'];
```

Comment:
One thing to note, you don't need to do a cast here to List<Contact>
When you query FROM a specific SObject, it will return a List of that SObject